operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -630,10 +631,31 @@ public boolean wasInspectionRun() {
 
     private boolean isShouldContinue() {
       DomainPresenceInfo cachedInfo = getExistingDomainPresenceInfo(getNamespace(), getDomainUid());
+      int currentRetryCount = Optional.ofNullable(liveInfo)
+          .map(DomainPresenceInfo::getDomain)
+          .map(Domain::getStatus)
+          .map(DomainStatus::getRetryCount)
+          .orElse(0);
+
+      String existingError = Optional.ofNullable(liveInfo)
+          .map(DomainPresenceInfo::getDomain)
+          .map(Domain::getStatus)
+          .map(DomainStatus::getMessage)
+          .orElse(null);
+
       if (cachedInfo == null || cachedInfo.getDomain() == null) {
         return true;
+      } else if (currentRetryCount > DomainPresence.getDomainPresenceFailureRetryMaxCount()) {
+        LOGGER.fine("Stop introspection retry - exceeded configured domainPresenceFailureRetryMaxCount: "
+            + DomainPresence.getDomainPresenceFailureRetryMaxCount());
```

Comment:
Is it only introspection failures that cause a limited number of retries, or there other types of validation failures?  If so, then the wording is misleading.

Should the text say 'domain.spec.retryCount' instead of 'domainPresenceFailureRetryMaxCount'?

Reply:
The max count actually is an operator tuning parameter not in domain spec